test/gtest/ucp/test_ucp_tag_xfer.cc

```diff
@@ -186,7 +183,7 @@ void test_ucp_tag_xfer::test_xfer(xfer_func_t func, bool expected, bool sync,
         if (!expected) {
             count = ucs_min(count, 50);
         }
-        ms << count << "x10^" << i << " " << std::flush;
+        UCS_TEST_MESSAGE << count << "x10^" << i << " " << std::flush;
```

Comment:
will it print every size separately with `INFO` tag now? Before it was like:

`[     INFO ] 0 1581x10^1 500x10^2 158x10^3 50x10^4 15x10^5 5x10^6 3x10^7`

Reply:
yes, you're right
```
[----------] 116 tests from udx/test_ucp_tag_xfer
[ RUN      ] udx/test_ucp_tag_xfer.contig_unexp/1 <ud_x>
[     INFO ] 0 
[     INFO ] 50x10^1 
[     INFO ] 50x10^2 
[     INFO ] 50x10^3 
[     INFO ] 50x10^4 
[     INFO ] 15x10^5 
[     INFO ] 5x10^6 
[     INFO ] 3x10^7 
[       OK ] udx/test_ucp_tag_xfer.contig_unexp/1 (3652 ms)
```

need to fix it